File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/dto/response/CustomBookingResponse.kt
package com.pickleballcourtbookingsystem.api.dtos

import android.os.Parcelable
import com.maxholmes.androidapp.data.model.Address
import com.maxholmes.androidapp.data.model.Booking
import com.maxholmes.androidapp.data.model.CourtTimeSlot
import kotlinx.parcelize.Parcelize

@Parcelize
data class CustomBookingResponse(
    val booking: Booking? = null,
    val courtClusterName: String? = null,
    val courtNumber: Int? = null,
    val address: Address? = null,
    val courtTimeSlots: List<CourtTimeSlot>? = null,
    val courtOwnerPhoneNumber: String? = null,
    val customerPhoneNumber: String? = null,
    val lastUpdatedTime: String
) : Parcelable

File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/DTOs/CustomBookingResponse.cs
using PickleballCourtBookingSystem.Core.Entities;
namespace PickleballCourtBookingSystem.Core.DTOs;

public class CustomBookingResponse
{
    public Booking? Booking { get; set; }
    public string? CourtClusterName { get; set; }
    public int? CourtNumber { get; set; }
    public Address? Address { get; set; }
    public List<CourtTimeSlot>? CourtTimeSlots { get; set; }
    public string? CourtOwnerPhoneNumber { get; set; }
    
    public string? CustomerPhoneNumber { get; set; }
    
    public DateTime LastUpdatedTime { get; set; }
}